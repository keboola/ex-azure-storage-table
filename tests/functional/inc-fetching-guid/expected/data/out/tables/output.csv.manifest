{
    "primary_key": [
        "id"
    ],
    "columns": [
        "id",
        "guid"
    ],
    "incremental": true
}
