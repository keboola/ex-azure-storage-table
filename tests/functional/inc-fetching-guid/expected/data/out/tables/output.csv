"1","2e4437ce-7a91-40a8-997a-69d8d18afefb"
"20","9af071f2-d1eb-448a-936b-16cbe370763a"
"300","8f57927d-f437-4868-af1c-2b4706fa1d28"
"4000","066c93c2-c535-466a-ad90-c38f039e6287"
"50000","be0f9a11-5c7c-4de8-9f04-cbc4a879ac31"
